platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationFileInstaller.java

```diff
@@ -236,6 +236,14 @@ private static ZipFileApplicationDetails getAppDetailsFromFeature(ZipFile zipFil
         }
 
         DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+
+        try {
+            dbFactory.setFeature("http://apache.org/xml/features/nonvalidating/load-dtd-grammar", false);
```

Comment:
We can create a future task to clean any and all of these string literals out of the codebase if we want.